.github/workflows/docker.yml

```diff
@@ -0,0 +1,68 @@
+name: Docker
+
+on:
+  push:
```

Comment:
Shall we also add `workflow_dispatch:` as a trigger to be able to manually redo? Or is the registry read-only for the same SHA or something?

Reply:
The manual run worked: https://github.com/MarcoIeni/cargo-http-registry/actions/runs/5053965210